integration/airflow/setup.cfg

```diff
@@ -46,7 +46,7 @@ deps = -r dev-requirements.txt
 	airflow-2.6.1: apache-airflow==2.6.1
 whitelist_externals = bash
 commands = bash -ec "if [[ ! -f $0/airflow.db ]]; then airflow db reset -y; fi" {envdir}
-	python -m pytest --cov=openlineage --junitxml=test-results/junit.xml
+	python -m pytest --cov=openlineage --junitxml=test-results/junit.xml -vv
```

Comment:
Is this a desired change or a leftover ?

Reply:
Leftover.